.circleci/config.yml

```diff
@@ -133,15 +120,3 @@ workflows:
           filters:
             branches:
               only: master
-
-  test_live:
-    triggers:
-      - schedule:
-          cron: "0 0 * * *"
-          filters:
-            branches:
-              only:
-                - master
-    jobs:
-      - test_live:
-          context: up42-py-live-testing
```

Comment:
Can we also delete this context?

Reply:
Or have a jira task to delete it

Reply:
good catch!
deleted from circleCI

Reply:
actually I should have waited for this PR to be merged to master before deleting.
But should be fine though